pkg/server/plugin/nodeattestor/aws/iid_test.go

```diff
@@ -237,9 +237,9 @@ func (s *IIDAttestorSuite) TestClientAndIDReturns() {
 			expectID:     "spiffe://example.org/spire/agent/aws_iid/test-account/test-region/test-instance",
 		},
 		{
-			desc:     "success, no client call, default template",
-			skipEC2:  true,
-			expectID: "spiffe://example.org/spire/agent/aws_iid/test-account/test-region/test-instance",
+			desc:      "success, no client call, default template",
+			allowList: []string{testAccount},
```

Comment:
wouldn't hurt to also have a test that asserts that a different account name in the allowlist does not result in `testAccount` being allowed